packages/@aws-cdk/aws-appsync/test/appsync-dynamodb.test.ts

```diff
@@ -1,15 +1,93 @@
 import '@aws-cdk/assert/jest';
-import { MappingTemplate, PrimaryKey, Values } from '../lib';
+import * as path from 'path';
+import * as db from '@aws-cdk/aws-dynamodb';
+import * as cdk from '@aws-cdk/core';
+import * as appsync from '../lib';
 
 function joined(str: string): string {
   return str.replace(/\s+/g, '');
 }
 
+// GLOBAL GIVEN
+let stack: cdk.Stack;
+let api: appsync.GraphQLApi;
+beforeEach(() => {
+  stack = new cdk.Stack();
+  api = new appsync.GraphQLApi(stack, 'baseApi', {
+    name: 'api',
+    schemaDefinition: appsync.SchemaDefinition.FILE,
+    schemaDefinitionFile: path.join(__dirname, 'appsync.test.graphql'),
+  });
+});
+
+describe('DynamoDb Data Source configuration', () => {
+  // GIVEN
+  let table: db.Table;
+  beforeEach(() => {
+    table = new db.Table(stack, 'table', {
+      partitionKey: {
+        name: 'id',
+        type: db.AttributeType.STRING,
+      },
+    });
+  });
+
+  test('default configuration produces name `DynamoDbCDKDataSource`', () => {
+    // WHEN
+    api.addDynamoDbDataSource('ds', table);
+
+    // EXPECT
```

Comment:
```suggestion
    // THEN
```

rationale: align with our convention elsewhere. few places in this PR that need to align to the convention